QuowMinimap.lua

```diff
@@ -262,12 +262,15 @@ if (string.sub(iMushclientVersion, -4) == "-pre") then
 end
 iMushclientVersion = tonumber(iMushclientVersion)
 
+bIsWindows = GetInfo(268) < 100
+
+sPathSep = bIsWindows and "\\" or "/"
+
 -- *******************************************************
 -- * Check for automatic \maps\ path in plugin location: *
 -- *******************************************************
 if (sQuowMapPath == "") then
-  sQuowMapPath = ".\\quow_plugins\\maps\\"
-  --sQuowMapPath = (GetPluginInfo(sMinimapID, 20)) .. "maps\\"
+  sQuowMapPath = "." .. sPathSep .. "quow_plugins" .. sPathSep .. "maps" .. sPathSep
```

Comment:
On non-Windows platforms, file paths must use forward slashes rather than backward slashes. So on Windows, `sQuowMapPath` is `".\\quow_plugins\\maps\\"`, but on other platforms, it's `./quow_plugins/maps/`.